llvm/test/Transforms/LoopVectorize/AArch64/sve-inductions-unusual-types.ll

```diff
@@ -9,26 +10,78 @@ target triple = "aarch64-unknown-linux-gnu"
 ; DEBUG: Found an estimated cost of Invalid for VF vscale x 1 For instruction:   %indvars.iv.next1295 = add i7 %indvars.iv1294, 1
 
 define void @induction_i7(ptr %dst) #0 {
-; CHECK-LABEL: @induction_i7(
+; CHECK-LABEL: define void @induction_i7(
+; CHECK-SAME: ptr [[DST:%.*]]) #[[ATTR0:[0-9]+]] {
```

Comment:
I think this test intentionally didn't use `update_test_checks.py`, so now there are a lot of additional lines in the diff.

Would it be possible to manually update the checks without too much effort? Otherwise it would be good to re-generate the check lines in a separate change first, so only functional changes are listed in the diff.

Reply:
Updated this test manually with only the changes needed for it to pass. The interleaving count changes for these test cases & it's unclear to me if that contradicts the original goal of the test.